config/crd/standard/gateway.networking.k8s.io_httproutes.yaml

```diff
@@ -1848,6 +1848,115 @@ spec:
                                       - 302
                                       type: integer
                                   type: object
+                                responseHeaderModifier:
```

Comment:
This PR should not result in a change to the standard set of CRDs. Once you get `<gateway:experimental>` tags in all the right places, this diff should disappear.